gpt_engineer/main.py

```diff
@@ -20,6 +20,9 @@
 def load_env_if_needed():
     if os.getenv("OPENAI_API_KEY") is None:
         load_dotenv()
+    if os.getenv("OPENAI_API_KEY") is None:
+        # if there is no .env file, try to load from the current working directory
+        load_dotenv(dotenv_path=os.getcwd() + "/.env")
```

Comment:
Please use:
load_dotenv(dotenv_path=os.path.join(os.getcwd(), ".env"))
instead of direct concatenation.

Reply:
Done in the last commit, thanks for the hint.